content/100-getting-started/01-quickstart-prisma-postgres.mdx

```diff
@@ -0,0 +1,193 @@
+
+Once the green **`CONNECTED`** label appears, your database is ready to use!
+
+## 2. Download example and install dependencies
+
+Copy the `try-prisma` command that's shown in the Console UI, paste it into your terminal and execute it.
+
+For reference, this is what the command looks like (note that the `__YOUR_DATABASE_CONNECTION_STRING__` placeholder must be replaced with _your_ actual database connection string):
+
+```
+npx try-prisma@latest
+  --template prisma-starter
+  --connection-string __YOUR_DATABASE_CONNECTION_STRING__
+  --name hello-prisma-postgres
+  --install npm
+  --path .
+```
+
+Your connection string that should replace the `__YOUR_DATABASE_CONNECTION_STRING__` placeholder looks similar to this: `prisma+postgres://accelerate.prisma-data.net/?api_key=ey...`
+
+Once the `try-prisma` command has terminaed, navigate into the project directory and install dependencies:
+
+```
+cd hello-prisma-postgres
+npm install
+```
+
+## 3. Create database tables (with a schema migration)
+
+Next, you need to create the tables in your database. You can do this by creating and executing a schema migration with the following command of the Prisma CLI:
+
+```
+npx prisma migrate dev --name init
+```
+
+This will map the `User` and `Post` models that are defined in your [Prisma schema](/orm/prisma-schema/) to your database. You can also review the SQL migration that was executed and created the tables in the newly created `prisma/migrations` directory.
+
+## 4. Execute queries with Prisma ORM
+
+The [`src/queries.ts`](./src/queries.ts) script contains a number of CRUD queries that will write and read data in your database. You can execute it by running the following command in your terminal:
+
+```
+npm run queries
+```
+
+Once the script has terminated, you can inspect the logs in your terminal or use Prisma Studio to explore what records have been created in the database:
+
+```
+npx prisma studio
+```
+
+## 5. Explore caching with Prisma Accelerate
+
+The [`src/caching.ts`](./src/caching.ts) script contains a sample query that uses [Stale-While-Revalidate](/accelerate/caching#stale-while-revalidate-swr) (SWR) and [Time-To-Live](/accelerate/caching#time-to-live-ttl) (TTL) to cache a database query using Prisma Accelerate. You can execute it as follows:
+
+```
+npm run caching
+```
+
+Take note of the time that it took to execute the query, e.g.:
+
+```
+The query took 2009.2467149999998ms.
+```
+
+Now, run the script again:
+
+```
+npm run caching
+```
+
+You'll notice that the time the query took will be a lot shorter this time, e.g.:
+
+```
+The query took 300.5655280000001ms.
+```
+
+## 6. Observe real-time events in your database
+
+The  [`src/realtime.ts`](./src/realtime.ts) script contains a demo for receiving real-time change [events](/pulse/database-events) from your database. You can start the script as follows:
+
+```
+npm run realtime
```

Comment:
Unrelated to docs, I remember running this, and not realizing that it's only subscribed to the User table. Maybe we can make that more clear in the terminal output, if it's still the case that it's not crystal clear?